src/android/com/microsoftopentechnologies/intellij/forms/ManageSubscriptionForm.form

```diff
@@ -1,30 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-Copyright 2014 Microsoft Open Technologies, Inc.
- 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
- 
-http://www.apache.org/licenses/LICENSE-2.0
- 
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
--->
 <form xmlns="http://www.intellij.com/uidesigner/form/" version="1" bind-to-class="com.microsoftopentechnologies.intellij.forms.ManageSubscriptionForm">
```

Comment:
The designer file associated to forms is auto-generated, and removes the copyright notice with any small change, making it difficult to track and maintain.